container/image_test.py

```diff
@@ -281,16 +281,28 @@ def test_build_with_tag(self):
 
   def test_with_passwd(self):
     with TestImage('with_passwd') as img:
-      self.assertDigest(img, '9fd1e04ef3070dc98b3b305e9e6091fd9e5fbf4a36d0928b5798a2d9ec2508ba')
+      self.assertDigest(img, '7de06a352108df247a4b5a1cea2eb3e80ee39ddc1192decbca8038427b4a2da6')
       self.assertEqual(1, len(img.fs_layers()))
-      self.assertTopLayerContains(img, ['.', './etc', './etc/passwd'])
+      self.assertTopLayerContains(img, ['.', './tmp', './tmp/passwd'])
 
       buf = cStringIO.StringIO(img.blob(img.fs_layers()[0]))
       with tarfile.open(fileobj=buf, mode='r') as layer:
-        content = layer.extractfile('./etc/passwd').read()
+        content = layer.extractfile('./tmp/passwd').read()
         self.assertEqual(
           'foobar:x:1234:2345:myusernameinfo:/myhomedir:/myshell\n', content)
 
+  def test_with_passwddefault(self):
+    with TestImage('with_passwddefault') as img:
+      self.assertDigest(img, '7de06a352108df247a4b5a1cea2eb3e80ee39ddc1192decbca8038427b4a2da6')
```

Comment:
This matches the digest above, but has different content, so I'm pretty sure (without looking) that this is a reason your tests are failing.  I would expect at least one of these tests to match the digest on the LHS: `9fd1e04...`.